charts/traction/README.md

```diff
@@ -98,6 +98,7 @@ kubectl delete secret,pvc --selector "app.kubernetes.io/instance"=my-release
 | `acapy.argfile.yml.endorser-alias`                      | For transaction Authors, specify the alias of the Endorser connection that will be used to endorse transactions.                                                                                                                                                                                                                                                                                    | `endorser`                            |
 | `acapy.argfile.yml.endorser-protocol-role`              | Specify the role ('author' or 'endorser') which this agent will participate. Authors will request transaction endorement from an Endorser. Endorsers will endorse transactions from Authors, and may write their own  transactions to the ledger. If no role (or 'none') is specified then the endorsement protocol will not be used and this agent will write transactions to the ledger directly. | `author`                              |
 | `acapy.argfile.yml.endorser-public-did`                 | For transaction Authors, specify the public DID of the Endorser agent who will be endorsing transactions.                                                                                                                                                                                                                                                                                           | `Ket75eV5UQvVkW2XBjgDH7`              |
+| `acapy.argfile.yml.genesis-transactions-list`           | Path to YAML configuration for connecting to multiple HyperLedger Indy ledgers.                                                                                                                                                                                                                                                                                                                        | `/home/aries/ledgers.yml`             |
 | `acapy.argfile.yml.genesis-url`                         | Specifies the url from which to download the genesis transactions. For example, if you are using 'von-network', the URL might be 'http://localhost:9000/genesis'. Genesis transactions URLs are available for the Sovrin test/main networks.                                                                                                                                                        | `http://test.bcovrin.vonx.io/genesis` |
```

Comment:
Related to the other comment, `acapy.argfile.yml.endorser-public-did` and `acapy.argfile.yml.genesis-url` are not necessary anymore.